dataeng/resources/retirement-partner-report-cleanup.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -ex
+
+# Create and activate a virtualenv.  In case we ever change the concurrency
+# setting on the jenkins worker, it would be safest to keep the builds from
+# clobbering each other's virtualenvs.
+VENV="venv-${BUILD_NUMBER}"
+virtualenv --python=python3.8 --clear "${VENV}"
+source "${VENV}/bin/activate"
+
+# prepare credentials
+mkdir -p $WORKSPACE/user-retirement-secure
+cp $USER_RETIREMENT_SECURE_DEFAULT $WORKSPACE/user-retirement-secure/secure-default.yml
+
+# prepare tubular
+cd $WORKSPACE/tubular
+# snapshot the current latest versions of pip and setuptools.
+pip install 'pip==21.0.1' 'setuptools==53.0.0'
```

Comment:
@brianhw  sure, can do that.
Should I create a file in  [https://github.com/openedx/tubular](https://github.com/openedx/tubular) repo. Because generally the requirement.txt files goes into code repository.
What are your on thoughts on if I place these requirements in [https://github.com/openedx/tubular/blob/master/requirements/base.in](https://github.com/openedx/tubular/blob/master/requirements/base.in) ?

Reply:
It seems like pip and possibly setuptools is considered "unsafe" to go into requirements.txt files: https://github.com/edx/tubular/blob/1552c897ad6d6423bd50bd1053fb95ce11b7d303/requirements/pip-tools.txt#L16-L17